prickly-pufferfish/exec/exec/problems/generator.py

```diff
@@ -0,0 +1,66 @@
+import json
+import random
+import string
+from pprint import pprint
+
+
+class typing:
+    pass
+
+
+class Generator:
+    """
+    a class for generating testcases
+    """
+
+    def __init__(self):
+        pass
+
+    def generate(self, func_name: str, types: list, num=10):
```

Comment:
Is this an exact duplicate of the file at `prickly-pufferfish/exec/exec/problems/permutations/generator.py`?  If so why have both?